aws/manageDeployment.sh

```diff
@@ -111,58 +111,6 @@ function register_resource_providers() {
   done
```

Comment:
This file should be moved under /execution?

Reply:
this is a script used by a human, so shouldn't got under `/execution` 
scripts in /execution essentially shouldn't be a command in the cli. where commands in `/aws` are commands that should have a cli command